tests/rbd/rbd_utils.py

```diff
@@ -919,3 +945,38 @@ def verify_migration_commit(rbd, pool, image):
     else:
         log.error((f"Image {image} is not found in pool {pool}"))
         raise ImageNotFoundError("Image Not found.")
+
+
+def verify_namespace_create(rbd, pool, namespace):
+    """verify namespace creation
+
+    Args:
+        rbd: rbd_object
+        pool: pool name
+    """
+    log.info("verifying namespcae creation")
+    out = rbd.exec_cmd(cmd=f"rbd namespace ls --pool {pool} --format json", output=True)
+    result = json.loads(str(out))
+    for value in result:
+        if value["name"] == namespace:
+            log.info(f"namespace {namespace} is successfully created")
+            return 0
+    log.error(f"namespace {namespace} creation failed")
+    return 1
```

Comment:
This method us just validating whether namespace exists in namespace list or not
IMO, we could generalise the method name and doc
We could use the method anytime, not just right after the namespace creation

Reply:
Done. Modified the method name, doc and log messages.